itou/templates/apply/includes/eligibility_diagnosis.html

```diff
@@ -1,7 +1,7 @@
 {% if siae.is_subject_to_iae_rules or request.user.is_prescriber %}
     {% if eligibility_diagnosis and eligibility_diagnosis.is_considered_valid %}
         <hr class="my-4">
-        <h3>{{ is_sent_by_authorized_prescriber|yesno:"Éligibilité à l'IAE,Critères d'éligibilité" }}</h3>
+        <h3>Éligibilité à l'IAE</h3>
         <p class="text-success mb-1">
             {{ is_sent_by_authorized_prescriber|yesno:"Confirmée,Validés" }}
```

Comment:
```suggestion
            {{ is_sent_by_authorized_prescriber|yesno:"Confirmée,Validée" }}
```

Pour ne pas trop faire de versions intermédiaires, c'est bien de garder cette règle, mais il vaut mieux faire l'accord pour ne pas écorcher les yeux des utilisateurs 😬